prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    div {
+      flex-grow: 6;
+      flex-basis: calc(100% / 6);
+    }
```

Comment:
Hi Jeff, rather than applying these declarations to ever div on the page, try just applying them to your card wrappers (`.card-wrapper`). This fix will need to be applied to all of your HTML files.